cdk/lib/cdk-stack.ts

```diff
@@ -7,7 +7,7 @@ export class CdkStack extends cdk.Stack {
     super(scope, id, props);
 
     // The code that defines your stack goes here
-    const myBucket = new s3.Bucket(this, "eo-lab-website", {
+    const myBucket = new s3.Bucket(this, process.env.DASHBOARD_BUCKET || "eo-lab-website", {
```

Comment:
i think when you deployed, the bucket changed and the last deployment is no longer available? [previous deployment](http://cdkstack-eolabwebsitebee4802d-2rjml2o8bbz3.s3-website-us-east-1.amazonaws.com/) says access denied now.

we might need a new way to manage deployments